### PR TITLE
Cleanup docs for targetFrameRate.

### PR DESCRIPTION
**Pull request by emackey**

### CODE REVIEW

File: Source/Widgets/Viewer/Viewer.js [hunk 1]
```diff
@@ -914,8 +914,8 @@ Either specify options.terrainProvider instead or set options.baseLayerPicker to
         /**
          * Gets or sets the target frame rate of the widget when <code>useDefaultRenderLoop</code>
          * is true. If undefined, the browser's {@link requestAnimationFrame} implementation
-         * determines the frame rate.  This value must be greater than 0 and a value higher than
-         * the underlying requestAnimationFrame implementatin will have no effect.
+         * determines the frame rate.  If defined, this value must be greater than 0.  A value higher
+         * than the underlying requestAnimationFrame implementation will have no affect.
```
Comment (shunter) [inline Source/Widgets/Viewer/Viewer.js]: "effect" is the correct spelling.

Comment (emackey) [inline Source/Widgets/Viewer/Viewer.js]: Yeah, not sure how that got changed, sorry!

Comment (shunter) [inline Source/Widgets/Viewer/Viewer.js]: copy/pasted from the doc in the other file, I assume, which is also wrong.

Comment (emackey) [inline Source/Widgets/Viewer/Viewer.js]: Yep that's what happened.  It was right in one copy and wrong in the other copy, and I had edited stuff in the other one, overwriting the one that was already correct.

**Comment by emackey** [comment]
Ready

**Pull request merged**